src/fillTank.test.js

```diff
@@ -1,11 +1,134 @@
 'use strict';
 
+const { fillTank } = require('./fillTank');
+
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  it('should fill the tank with the maximum amount if no amount is given',
+    () => {
+      const customer = {
+        money: 3000,
+        vehicle: {
+          maxTankCapacity: 40,
+          fuelRemains: 8,
+        },
+      };
+      const fuelPrice = 1.5;
+
+      fillTank(customer, fuelPrice);
+      expect(customer.vehicle.fuelRemains).toEqual(40);
+      expect(customer.money).toEqual(2952);
+    });
+
+  it('should fill the tank with the max amount if no amount is given',
```

Comment:
This test case is a duplicate of the first one, as it has the same setup and expectations. Consider removing it or changing the input/expectations to test a different case.